Description: Skip tests using upstream provided x86_64 binaries without source code.
 These binaries are non DFSG compatible and these tests fails on non x86_64 architectures.
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/990382

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -85,23 +85,29 @@
 mod tests {
     const LIB_RS: &'static str = "./src/lib.rs";
     const CARGO_FILE: &'static str = "./Cargo.toml";
+/*
     const RUST_BIN_FILE: &'static str = "./testdata/rust_hello";
     const C_BIN_FILE: &'static str = "./testdata/c_hello";
     const C_BIN_FILE_COPY: &'static str = "./testdata/c_hello_copy";
+*/
 
     #[test]
     fn diff_the_same_text_file() { assert!(super::diff(LIB_RS, LIB_RS)); }
 
+/*
     #[test]
     fn diff_the_same_binary_file() { assert!(super::diff(RUST_BIN_FILE, RUST_BIN_FILE)); }
 
     #[test]
     fn diff_identical_binary_files() { assert!(super::diff(C_BIN_FILE, C_BIN_FILE_COPY)); }
+*/
 
     #[test]
     fn diff_different_text_files() { assert!( !super::diff(LIB_RS, CARGO_FILE)); }
 
+/*
     #[test]
     fn diff_different_binary_files() { assert!( !super::diff(RUST_BIN_FILE, C_BIN_FILE)); }
+*/
 }
 
